conf/bridge-server.conf

```diff
@@ -151,3 +151,8 @@ local.study.whitelist = api
 dev.study.whitelist = api
 uat.study.whitelist = api,ios-sdk-int-tests,asthma,cardiovascular
 prod.study.whitelist = api,asthma,breastcancer,cardiovascular,diabetes,fphs,fphs-lab,ohsu-molemapper,parkinson,parkinson-lux,lilly
+
+local.usersigned.consents.bucket = dummy-value
```

Comment:
Would it be worthwhile to create a default local bucket in our S3 account, to reduce the amount of required work to bootstrap a new dev environment?

Reply:
Added local.usersigned.consents.bucket and reference it in the config